src/network/networkd-manager.c

```diff
@@ -413,7 +413,7 @@ int manager_rtnl_process_route(sd_netlink *rtnl, sd_netlink_message *message, vo
                 break;
 
         default:
-                log_link_debug(link, "rtnl: ignoring unsupported address family: %d", family);
+                assert_not_reached("Received unsupported address family");
```

Comment:
hmm, is it really the right thing to do here? assert_not_reached() aborts the program when we enter this piece of code. The old code just logged and ignored things. Are you sure it couldn't happen IRL that we get a family here we don't know?

Reply:
@poettering Please look line 349.

Reply:
I think we should not aborting in this case or other such like  cases as this family is sent by the kernel. We just log them and ignore.

Reply:
The line 349 ensures that family is `AF_INET` or `AF_INET6`. So, certainly the default entry is not reached.

Reply:
yeah that will never reach there we already filtered the family.